.travis.yml

```diff
@@ -1,31 +1,58 @@
-#This file is generated by ModuleSync, do not edit.
 ---
 sudo: false
+dist: trusty
 language: ruby
 cache: bundler
-script: "bundle exec rake release_checks"
+before_install:
+  - bundle -v
+  - rm Gemfile.lock || true
+  - gem update --system
+  - gem --version
+  - bundle -v
+script:
+  - 'bundle exec rake $CHECK'
+bundler_args: --without system_tests
+rvm:
+  - 2.4.1
+  - 2.1.9
+env:
+  - PUPPET_GEM_VERSION="~> 4.0" CHECK=spec
```

Comment:
This is testing Puppet 4.0 with 2.4.1 and PUppet 5.0 with 2.1.9.  This seems a waste.

Note Puppet 4 with 2.1.9 is still being tested twice